.github/workflows/build-coop.yaml

```
@@ -8,81 +8,57 @@ on:
+
+              - name: macOS_Intel
+                os: macos-15-intel
+                deps: brew install make mingw-w64 pkg-config glfw3 coreutils
+                make: gmake OSX_BUILD=1 -j$(sysctl -n hw.ncpu)
+                internal: .app/Contents/MacOS/sm64coopdx
+                zip: sm64coopdx.app
+            include:
+              - platform:
+                  - make: make -j$(nproc)
+                
+        name: ${{ matrix.plat.name }}
```

actionlint (1.7.12)

[error] property "plat" is not defined in object type {platform: any}

(expression)